spec/menu-manager-spec.coffee

```diff
@@ -62,7 +62,7 @@ describe "MenuManager", ->
       atom.keymaps.add 'test', 'atom-workspace': 'ctrl-b': 'b'
       menu.update()
 
-      waits 1
+      waits 50
```

Comment:
Appveyor failed twice on this spec so I bumped the waits to be consistent with the other tests in this file and now it passes.